recipes/nibabel/run_test.py

```diff
@@ -0,0 +1,14 @@
+import nose
+import sys
+import re
+
+splatform = sys.platform
+if splatform.startswith('win32'):
+    # Installation of data files for script testing a bit rickety on Appveyor:
+    config = nose.config.Config(verbosity=2,
+                                exclude=[re.compile("test_scripts"),
+                                         re.compile("test_dft")])
+else:
+    config = nose.config.Config(verbosity=2)
```

Comment:
> Failures remain only on Python 3.5/Windows. Any ideas?

It looks like there are genuine failures (You can E and F in the nosetests output) but for some reason it looks like the AppVeyor run was stopped before completing (there is no nosetests summary output showing the actual failures). Can you try doing a `git commit --amend` + force-push to trigger a new run of the CIs and double-check whether this was not a temporary glitch?